src/lang/en.json

```diff
@@ -237,6 +237,8 @@
     "modalPhrasePrintPreview.saveThisSheet": "Save this sheet! Print it and keep it in a safe place! This is your property!",
     "modalPhrasePrintPreview.title": "Print Preview",
     "modalPhrasePrintPreview.weCanNotChagneYourPassword": "We **CAN NOT** change your password. Please **DO NOT FORGET** to save your password. It is your private key. You will need this **Password + Mnemonic Phrase** to access your wallet.",
+    "modalPurchaseHbar.carbon": "Carbon",
```

Comment:
This should not be in the lang file as its a proper name

Reply:
removed it in favor of the logo